app/views/banks/show.json.jbuilder

```diff
@@ -0,0 +1,18 @@
+json.tel_num "#{@bank.tel_area_code}-#{@bank.tel_prefix}-#{@bank.tel_suffix}"
+json.address do
+  json.city @bank.city.split(" ").map { |el| el.downcase.capitalize}.join(" ")
+  json.state @bank.state_code
+  json.extract! @bank, :zipcode, :zipcode_ext
+  json.street_address @bank.address
+end
+json.last_change_date @bank.last_change_date.to_s[0...10]
```

Comment:
Since this is the view layer, I might opt for creating a more direct interface for the bank object instead of manipulating data here.